cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -147,11 +129,14 @@ func NewOptions(out, errOut io.Writer) *Options {
 		schema.GroupKind{Group: gardencorev1alpha1.GroupName},
 		schema.GroupKind{Group: gardencorev1beta1.GroupName},
 	)
+	apiserver.RegisterAllAdmissionPlugins(o.Recommended.Admission.Plugins)
+	o.Recommended.Admission.RecommendedPluginOrder = apiserver.AllOrderedPlugins
```

Comment:
Before, we only appended to the `RecommendedPluginOrder` and now there is a hard coded list only. Can this be a problem if new admission plugins are added through the API Server lib?

Reply:
Yes, we won't notice, when the default list of plugins in the API server library will change, that's correct.
Though, I would argue, that there is no way of correctly detecting automatically in which order we should insert our admission plugins between the defaults.

So, I added a small test, that will break, when the default set and/or order of admission plugins changes.
PTAL :)